app/services/app/app.ts

```diff
@@ -56,48 +56,46 @@ export class AppService extends StatefulService<IAppState>
 
   @track('app_start')
   load() {
-    // This is synchronous and can take a really long time for large configs.
-    // Setting a timeout allows the spinner and loading text to be drawn to
-    // the screen before starting on the slow synchronous operation.
-    // TODO: loading should be async
-    setTimeout(() => {
```

Comment:
This `setTimeout` was completely unnecessary and I think was a relic of refactoring.  The actually useful `setTimeout` is in the `loadConfig` function.